### PR TITLE
Theme updates

### PR DESCRIPTION
**Pull request by rrigdon**
Updated the theme to be on brand with our [Distribution site.](https://docs.anaconda.com/)

### CODE REVIEW

File: docs/_static/css/custom.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
+@import url("theme.css");
+
+.wy-nav-content {
```
Comment (soapy1) [inline docs/_static/css/custom.css]: Could you comment these to say what ui elements they change

File: docs/_static/css/custom.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+.wy-nav-side {
+/* This relates to the entire color of the sidebar */
+   background-color:#EEEEEE;
+}
+
+.wy-menu-vertical {
+    color: black;
+}
+
+h1 {
```
Comment (soapy1) [inline docs/_static/css/custom.css]: Could a class be attached to this?

**Comment by soapy1** [comment]
These changes look good to me! Is there a timeline of when this change should go in? Like do we want to do a design review?

**Comment by rrigdon** [comment]
It's been reviewed by Kelly and Laurel so I think it should be fine to publish this afternoon or first thing Monday.

**Pull request merged**